packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -262,17 +262,21 @@ contract Deploy is Deployer {
     /// @notice Deploy the DisputeGameFactoryProxy
     function deployDisputeGameFactoryProxy() onlyDevnet broadcast() public returns (address) {
         address proxyAdmin = mustGetAddress("ProxyAdmin");
-        Proxy proxy = new Proxy({
-            _admin: proxyAdmin
-        });
 
-        address admin = address(uint160(uint256(vm.load(address(proxy), OWNER_KEY))));
-        require(admin == proxyAdmin);
+        string[2] memory contractNames = ["DisputeGameFactoryProxy", "MIPSDisputeGameFactoryProxy"];
+        for (uint256 i; i < contractNames.length; i++) {
+            Proxy proxy = new Proxy({
+                _admin: proxyAdmin
+            });
 
-        save("DisputeGameFactoryProxy", address(proxy));
-        console.log("DisputeGameFactoryProxy deployed at %s", address(proxy));
+            address admin = address(uint160(uint256(vm.load(address(proxy), OWNER_KEY))));
+            require(admin == proxyAdmin);
 
-        return address(proxy);
```

Comment:
These functions have a baked in assumption that they deploy a single contract based on the return signature, it will only impact `forge script --sig 'deployDisputeGameFactoryProxy() | jq .`